Sensus.iOS.Tests/AppDelegate.cs.txt

```diff
@@ -0,0 +1,60 @@
+// ***********************************************************************

```

Comment:
I'm not sure. All my template automatically included them. Not being 100% familiar with the templates I decided to leave them for now. We can always remove later.

As for the Resource.designer.cs files, good catch. They snuck in. I fixed the .gitignore rule after I'd commited a few apparently. I'll remove them.